Source/SteveSloth/Public/MyEnemyManager.h

```diff
@@ -0,0 +1,31 @@
+/****************************************************************************************
+* Copyright:
+* Name:
+* Script: MyEnemyManager.h
+* Date: 
+* Description: 
+* TODO: 
+* Known Bugs: 
+*****************************************************************************************/
```

Comment:
fill this out

Reply:
Sorry I have not even got into the enemymanager yet.. that was something richard and i were talking about and I created it then dropped it.. I will be picking it back up before the end of class..

Reply:
I still would say just fill out the top with basic info

Reply:
the actual script can stay empty for now

Reply:
All done